src/mockgyver_xform.erl

```diff
@@ -177,21 +177,26 @@ is_mock_expr(tuple, Form) ->
     case erl_syntax:tuple_elements(Form) of
         [H | T] ->
             case erl_syntax:is_atom(H, '$mock') of
-                true  -> {true, analyze_mock_form(T)};
-                false -> false
+                true  ->
+                    [TypeTree | Rest] = T,
+                    Type = erl_syntax:atom_value(TypeTree),
+                    Location = erl_syntax:concrete(lists:last(T)),
```

Comment:
Minor, use lists:last(Rest) and swap 183 and 184 (just to keep them in the other they come?

Reply:
will do

Reply:
Perfect, thanks!